docs/static/filebeat-modules.asciidoc

```diff
@@ -1,143 +1,124 @@
-[2017-10-13T00:01:15,443][INFO ][logstash.pipeline        ] Pipeline started {"pipeline.id"=>"main"}
-----------------------------------------------------------------------
+== Working with {filebeat} Modules
+
+{filebeat} comes packaged with pre-built
+{filebeat-ref}/filebeat-modules.html[modules] that contain the configurations
+needed to collect, parse, enrich, and visualize data from various log file
+formats. Each {filebeat} module consists of one or more filesets that contain
+ingest node pipelines, {es} templates, {filebeat} input configurations, and
+{kib} dashboards.
+
+You can use {filebeat} modules with {ls}, but you need to do some extra setup.
+The simplest approach is to <<use-ingest-pipelines,set up and use the ingest
+pipelines>> provided by {filebeat}. If the ingest pipelines don't meet your
+requirements, you can
+<<logstash-config-for-filebeat-modules,create {ls} configurations>> to use
+instead of the ingest pipelines.
+
+Either approach allows you to use the configurations, index templates, and
+dashboards available with {filebeat} modules, as long as you maintain the
+field structure expected by the index and dashboards.
 
+[[use-ingest-pipelines]]
+=== Use ingest pipelines for parsing
 
-The Logstash pipeline is now ready to receive events from Filebeat. Next, you
-set up and run Filebeat.
+When you use {filebeat} modules with {ls}, you can use the ingest pipelines
+provided by {filebeat} to parse the data. You need to load the pipelines
+into {es} and configure {ls} to use them.
 
-[float]
-==== Set up and run Filebeat
+*To load the ingest pipelines:*
+
+On the system where {filebeat} is installed, run the `setup` command with the
+`--pipelines` option specified to load ingest pipelines for specific modules.
+For example, the following command loads ingest pipelines for the system and
+nginx modules:
 
-. If you haven't already set up the Filebeat index template and sample Kibana
-dashboards, run the Filebeat `setup` command to do that now: 
-+
 [source,shell]
-----------------------------------------------------------------------
-./filebeat -e setup
-----------------------------------------------------------------------
-+
-The `-e` flag is optional and sends output to standard error instead of syslog.
-+
-A connection to Elasticsearch and Kibana is required for this one-time setup
-step because Filebeat needs to create the index template in Elasticsearch and
-load the sample dashboards into Kibana. 
-+
-After the template and dashboards are loaded, you'll see the message `INFO
-Kibana dashboards successfully loaded. Loaded dashboards`.
-
-. Configure Filebeat to send log lines to Logstash. To do this, in the
-+filebeat.yml+ config file, disable the Elasticsearch output, and enable the
-Logstash output. For example:
-+
-[source,yaml]
 -----
-#output.elasticsearch:
-  #hosts: ["localhost:9200"]
-output.logstash:
-  hosts: ["localhost:5044"]
+filebeat setup --pipelines --modules nginx,system
```

Comment:
For https://github.com/elastic/beats/issues/9037 we'd still need to add something in filebeat documentation.

Reply:
I'll create another PR that links to this doc after it's published.

Reply:
(marking this as resolved because I am tracking it in the beats issue.